### ISSUE TITLE
Images in 07-github do not correspond to text

### ISSUE THREAD

**Issue by scmbradley** [swcarpentry/git-novice#1033]
### What is the problem?

The text of the 07-github page refers to Alfredo Linguine's recipes repo, but the images in the page appear to be about user kekoziar's planets repo. Since the rest of the guide up to this point has been about recipes, the images should be adjusted to match.

### Location of problem (optional)

https://swcarpentry.github.io/git-novice/07-github.html

**Comment by scmbradley** [comment]
Yeah sure I can have a go. It'll be a few days before I get some time. Would the idea be to just take screenshots of GitHub from my user account at the relevant points of doing the steps in the page?

**Comment by martinosorb** [comment]
That would be fine, yes. Make sure no other personal info is visible. Thank you so much! If you change your mind please let us know so we can assign it to someone else.

**Comment by CFGrote** [comment]
I just stumbled across this, too. @scmbradley , are you working on a PR? I can offer my help if needed.

**Comment by martinosorb** [comment]
@CFGrote it's been a few weeks, so if you want to do it I'm sure it'll be fine. Thanks for offering.

**Comment by scmbradley** [comment]
@CFGrote hi, yes please. My son was born a little early, shortly after I opened this issue, so I have a lot less time than I expected.

**Comment by CFGrote** [comment]
OK,Im on it. Good luck @scmbradley and all best for your newborn

**Comment by erinmgraham** [comment]
Hi @CFGrote, I wanted to address part 3 in issue #900 and like most changes it was a loose string that was tied to this issue and the one mentioned by martinosorb above. Have you already started making these updates? If not, I'll assign this issue to myself.

**Comment by CFGrote** [comment]
yes, I'm on it, hoping to commit today. sorry for the delay

**Comment by erinmgraham** [comment]
Too easy! Thanks for the reply 🙂

________________________________
